CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree Android Drop-In Release Notes
 
+## unreleased
+
+* Filter Vaulted Payment Methods based on `DropInRequest` (fixes [#205](https://github.com/braintree/braintree-android-drop-in/issues/205))
```

Comment:
```suggestion
* Filter Vaulted Payment Methods based on `DropInRequest` (fixes #205)
```

As long as the issue is in the same repo, GitHub should make the link for you automatically!

Reply:
Good call. I always go back and forth with that haha. I like the less verbose way, it's a lot cleaner.